specification/resources/apps/models/app_log_destination_open_search_spec.yml

```
@@ -3,9 +3,8 @@ properties:
   endpoint:
     type: string
     description: >-
-      OpenSearch API Endpoint. Only HTTPS is supported. Format:
-      <host>:<port>. Default port is 9300.
-    example: "example.com:9300"
+      OpenSearch API Endpoint. Only HTTPS is supported. Format: https://\u003chost\u003e:\u003cport\u003e.
```

**Question:** Is the unicode characters necessary?

---

Curious about this as well. Was that intentional? 

```suggestion
      OpenSearch API Endpoint. Only HTTPS is supported. Format: `https://<host>:<port>`.
```

---

This is what is generated by our tooling.